File: Examples/test-suite/perl5/director_minimal_runme.pl
use strict;
use warnings;
use Test::More tests => 4;
BEGIN { use_ok 'director_minimal' }
require_ok 'director_minimal';

{
  package MyMinimal;
  use base 'director_minimal::Minimal';
  sub run { 1; }
}

my $c = MyMinimal->new();
isa_ok $c, 'MyMinimal';
is $c->get(), 1, 'return true';

File: Examples/test-suite/octave/director_minimal_runme.m
director_minimal

function self=MyMinimal()
  global director_minimal;
  self=subclass(director_minimal.Minimal());
  self.run=@MyMinimal_run;
end
function ret=MyMinimal_run(self)
  ret = true;
end

c = MyMinimal();
if (!c.get())
  error("Should return true");
end
